metrics/api/v1/api.go

```diff
@@ -16,20 +16,38 @@ package v1
 
 import (
 	restful "github.com/emicklei/go-restful"
+
 	"k8s.io/heapster/metrics/api/v1/types"
+	"k8s.io/heapster/metrics/core"
 	"k8s.io/heapster/metrics/sinks/metric"
 )
 
 type Api struct {
 	runningInKubernetes bool
 	metricSink          *metricsink.MetricSink
+	gkeMetrics          map[string]core.MetricDescriptor
+	gkeLabels           map[string]core.LabelDescriptor
 }
 
 // Create a new Api to serve from the specified cache.
 func NewApi(runningInKuberentes bool, metricSink *metricsink.MetricSink) *Api {
```

Comment:
It's an existing problem, but it wouldn't hurt to fix the "Kuberentes" typo while you're here